Translations/fr.po

```diff
@@ -5151,7 +5157,7 @@ msgstr "Zhar le Fou"
 #: Source/monstdat.cpp:476
 msgctxt "monster"
 msgid "Snotspill"
-msgstr "Déversement de Morve"
+msgstr "Morvure"
```

Comment:
Those name are hard to translate !

Reply:
I'm quite used to translate such tricky names, even with space constraints (10 letters or less) ;)